app/controllers/patients_controller.rb

```diff
@@ -56,6 +56,17 @@ def data_entry_create # temporary
       render 'data_entry'
     end
   end
+
+
+  def fetch_clinic_naf
+    accepts_naf =  Clinic.find_by(id: params['id']).accepts_naf
```

Comment:
You can just use `Clinic.find(params['id'])` here, no need to declare the field. A nice little rails-ism!